blockchain/fullblocktests/generate.go

```diff
@@ -2538,11 +2729,86 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 	g.AssertTipNumRevocations(2)
 	rejected(blockchain.ErrRevocationsMismatch)
 
+	// Create block that has a revocation with more payees than expected.
+	//   ... -> brt1(24)
+	//                  \-> brt3(25)
+	g.SetTip("brt1")
+	g.NextBlock("brt3", outs[25], ticketOuts[25], func(b *wire.MsgBlock) {
+		for idx := 0; idx < len(b.STransactions); idx += 1 {
+			if chaingen.IsRevocationTx(b.STransactions[idx]) {
+				b.STransactions[idx].TxOut = append(b.STransactions[idx].TxOut,
+					b.STransactions[idx].TxOut[0])
+				break
+			}
+		}
+	})
+	g.AssertTipNumRevocations(1)
+	rejected(blockchain.ErrSSRtxPayeesMismatch)
+
+	// Create block that has a recovation paying to less than the orignal
+	// amount to the committed address.
+	//   ... -> brt1(24)
+	//                  \-> brt4(25)
+	g.SetTip("brt1")
+	g.NextBlock("brt4", outs[25], ticketOuts[25], func(b *wire.MsgBlock) {
+		for idx := 0; idx < len(b.STransactions); idx += 1 {
+			if chaingen.IsRevocationTx(b.STransactions[idx]) {
+				b.STransactions[idx].TxOut[0].Value--
```

Comment:
This is not correct.  The consensus rules enforce revocations are not paying too much.  Nothing prevents them from paying less, which is what gives a fee to the miner to encourage them to include it.

It should be `Value++`.